packages/cashier/src/pages/withdrawal/withdrawal.tsx

```diff
@@ -123,11 +142,7 @@ const Withdrawal = observer(() => {
 
     if (is_crypto_transactions_visible) return <CryptoTransactionsHistory />;
 
-    return (
-        <PageContainer hide_breadcrumb right={<WithdrawalSideNotes />}>
```

Comment:
Move the `<PageContainer />` down into the `<WithdrawalPageContent />` since we don't want the side notes for `<WithdrawalVerificationEmail />`